src/Tasks/Microsoft.NET.Build.Tasks.UnitTests/GivenThatWeHaveErrorCodes.cs

```diff
@@ -37,6 +37,7 @@ public class GivenThatWeHaveErrorCodes
             1066,
             1101,
             1108,
+            1180,
```

Comment:
Is the a corresponding code change for the deletion?

Reply:
No, I think that none of these unit tests are running in CI, so this happened earlier and I just noticed when I ran the test manually.